### DISCUSSION TITLE
Preparing Editor Functionality 很慢

### DISCUSSION THREAD

**Discussion by GanZhiXiong** [qwertyyb/Fire#82] [General]
<img width="447" alt="image" src="https://user-images.githubusercontent.com/27394631/184627338-1f466316-8ca5-4e63-a7ad-afaa88bda324.png">

**Comment by ShikiSuen** [comment]
這是 Xcode 的現狀。Xcode 在組織 SwiftUI 內容預覽時就會這樣慢。
估計 Apple 不太可能會再針對 intel mac 做多少優化了。

P.S.: Xcode 14 比 Xcode 13 快。
